resources/windows_ad_domain_forest_2008.rb

```diff
@@ -0,0 +1,84 @@
+
+require 'mixlib/shellout'
+
+resource_name :windows_ad_domain_forest_2008
```

Comment:
This resource file should be named domain_forest_2008, which would also resolve the need for this line. The resource files shouldn't contain the cookbook name.